.github/workflows/zscaler-iac-scan.yml

```
@@ -47,10 +70,12 @@ jobs:
           #To fail the build based on policy violations identified in the IaC templates, set the input value (fail_build) to true.
-          fail_build : #Enter true/false
+          # Report rather than block, matching how trivy.yml treats scanners on
+          # pull requests. Enforcement belongs in a deliberate main-only job.
+          fail_build : 'false'
       #Ensure that the following step is included in order to post the scan results under the code scanning alerts section within the repository.
       - name: Upload SARIF file
-        if: ${{ success() || failure() && (steps.zscaler-iac-scan.outputs.sarif_file_path != '') }}
+        if: ${{ steps.credentials.outputs.configured == 'true' && (success() || failure()) && (steps.zscaler-iac-scan.outputs.sarif_file_path != '') }}
         uses: github/codeql-action/upload-sarif@v3
         with:
           sarif_file: ${{ steps.zscaler-iac-scan.sarif_file_path }}
```

actionlint (1.7.12)

[error] property "sarif_file_path" is not defined in object type {conclusion: string; outcome: string; outputs: {string => string}}

(expression)